Build cri-o failed because of golang version is not match

<!--
If you are reporting a new issue, make sure that we do not have any duplicates
already open. You can ensure this by searching the issue list for this
repository. If there is a duplicate, please close your issue and add a comment
to the existing issue instead.

If you suspect your issue is a bug, please edit your issue description to
include the BUG REPORT INFORMATION shown below. If you fail to provide this
information within 7 days, we cannot debug your issue and will close it. We
will, however, reopen it if you later provide the information.

For more information about reporting issues, see
https://github.com/cri-o/cri-o/blob/master/CONTRIBUTING.md#reporting-issues

---------------------------------------------------
GENERAL SUPPORT INFORMATION
---------------------------------------------------

The GitHub issue tracker is for bug reports and feature requests.
General support for **CRI-O** can be found at the following locations:

- IRC - #cri-o channel on irc.freenode.org
- Slack - kubernetes.slack.com #sig-node channel
- Post a question on StackOverflow, using the CRI-O tag

---------------------------------------------------
BUG REPORT INFORMATION
---------------------------------------------------
Use the commands below to provide key information from your environment:
You do NOT have to include this information if this is a FEATURE REQUEST
-->

**Description**

<!--
Briefly describe the problem you are having in a few paragraphs.
-->

**Steps to reproduce the issue:**
1.Install golang 1.10.2 on  amd64 ubuntu 16.04
2.`go get -u github.com/cri-o/cri-o`
3.`cd ~/goPath/src/github.com/cri-o/cri-o/ `
4.`make`

**Describe the results you received:**
```
go build -ldflags '-s -w -X main.gitCommit="66eaece433489575f46261b90252c84c58b7e93e" -X main.buildInfo=1559785299' -tags "containers_image_ostree_stub     libdm_no_deferred_remove  seccomp selinux" -o bin/crio github.com/cri-o/cri-o/cmd/crio
# github.com/cri-o/cri-o/oci
oci/runtime_vm.go:175:20: undefined: strings.ReplaceAll
Makefile:120: recipe for target 'bin/crio' failed
make: *** [bin/crio] Error 2
```

**Describe the results you expected:**
can build cri-o successfully
**Additional information you deem important (e.g. issue happens only occasionally):**
 I follow the setup.md building the cri-o.The doc indicated that I should use golang `1.8.x`or higher.I use golang 1.10.2 but build failed.
 Can we update on doc about the accurate golang version?

--- a/tutorials/setup.md
+++ b/tutorials/setup.md
@@ -69,7 +69,7 @@ apt-get update -qq && apt-get install -y \
 
 If using an older release or a long-term support release, be careful to double-check that the version of `runc` is new enough (running `runc --version` should produce `spec: 1.0.0`), or else build your own.
 
-Be careful to double-check that the version of golang is new enough, version 1.8.x or higher is required.  If needed, golang kits are available at https://golang.org/dl/
+Be careful to double-check that the version of golang is new enough, version 1.12.x or higher is required.  If needed, golang kits are available at https://golang.org/dl/
 
 ## Get Source Code
 

